### PR TITLE
[Doc]Update Get Started Guide

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix a part of https://github.com/apache/bookkeeper/issues/2193.

I've updated this chapter based on the following architecture.
image

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@sijie @eolivelli could you please help review? Thank you

**Comment by Anonymitaet** [comment]
@sijie could you please help review? Thank you

File: site/docs/latest/getting-started/run-locally.md [hunk 1]
```diff
@@ -1,16 +1,115 @@
...
-This would start up an ensemble with 10 bookies:
+## Install BookKeeper standalone
+
+Before installation, you need to check system requirements.
+
+### System requirement
+
+* [Unix environment](http://www.opengroup.org/unix)
+  
+* [Java Development Kit 1.6](http://www.oracle.com/technetwork/java/javase/downloads/index.html) or later
```
Comment (eolivelli) [inline site/docs/latest/getting-started/run-locally.md]: Replace 1.6 with 8
I would link to https://adoptopenjdk.net as Oracle jdk is no more free for production usage

Comment (Anonymitaet) [reply site/docs/latest/getting-started/run-locally.md]: @eolivelli sorry for the later reply, I saw your message just now.
Many thanks for your comment, I've incorporated it, PTAL.

**Comment by shoothzj** [comment]
closed by no updates

**Pull request closed without merging**